app/models/post.rb

```diff
@@ -1,2 +1,5 @@
 class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  has_many :comments
+  has_many :likes
```

Comment:
- [ ] as part of the requirements the Post model should have :
- a method that updates the posts counter for a user
- a method that returns the 5 most recent comments for a given post

kindly add that :pray: